lib/context.js

```diff
@@ -1,13 +1,17 @@
 import { readFile, writeFile } from 'mz/fs'
 import { resolve } from 'path'
 import { homedir } from 'os'
+import { sync } from 'find-up'
 
 import { proxyStringToObject } from './utils/proxy'
 
 let context
 
 export function getConfigPath () {
-  return resolve(homedir(), '.contentfulrc.json')
+  const contentfulrc = '.contentfulrc.json'
+  const configPath = sync(contentfulrc)
```

Comment:
Did you investigate if we can make all the related functions async? I know I said we could do it sync or even have to in current state but I'd prefer using the async apis of our dependencies cus they are in theory more performant.

Reply:
yea there are async apis!